microsoft-365/compliance/gdpr-arc-Azure.md

```diff
@@ -86,8 +86,8 @@ For more GDPR-related documentation, visit [https://aka.ms/gdprgetstarted](https
 | ***Information Security Policies (6.2)*** | The customer should augment any existing information security policies to include protection of personal data, including policies necessary for compliance with any applicable legislation. | Microsoft policies for information security and any specific measures for the protection of personal information.<br>- *Microsoft Azure (All-Up) ISO/IEC 27001:2013 ISMS Statement of Applicability* [13] see A.19-A.29<br>- SOC 2 Type 2 Audit Report [[12](gdpr-arc-Azure.md#12)] | (24)(2) |
-| ***Organization of Information Security Customer consideration(6.3)*** | The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles. | An overview of the role of Microsoft's Data Protection Officer, the nature of his duties, reporting structure and contact information.<br>- Microsoft DPO Information [[17](gdpr-arc-Azure.md#17)] | (37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2) |
-| ***Human Resource Security (6.4)*** | The customer should determine and assign responsibility for providing relevant training related to protecting personal data. | An overview of the role of Microsoft's Data Protection Officer, the nature of his duties, reporting structure and contact information.<br>- *Microsoft Cloud Security Policy* [[4](gdpr-arc-Azure.md#4)] see 03. Human Resources Security<br>- *FedRAMP Moderate FedRAMP System Security Plan* [[3](gdpr-arc-Azure.md#3)] see 13.2 "Awareness and Training" (AT) | (39)(1)(b) |
+| ***Organization of Information Security Customer consideration(6.3)*** | The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles. | An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure and contact information.<br>- Microsoft DPO Information [[17](gdpr-arc-Azure.md#17)] | (37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2) |
```

Comment:
```suggestion
| ***Organization of Information Security Customer consideration(6.3)*** | The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles. | An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure, and contact information.<br>- Microsoft DPO Information [[17](gdpr-arc-Azure.md#17)] | (37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2) |
```